server/src/clientglue.js

```diff
@@ -21,9 +22,7 @@ exports.setModel = function (data) {
 
 function render() {
   setGitRevision(model.gitRevision);
-  let attrs = {
-    staticLink: staticLink
-  };
+  let attrs = { staticLink, ready, signUp, signIn, getProfile, updateProfile };
```

Comment:
I don't think these need to be properties?  staticLink is a special case because it is used on the server and is request-specific.  But the rest are only used on the client and can be called globally.  So frame.js can just include these functions and call them.